examples/tts/conf/zh/fastpitch_align_22050.yaml

```diff
@@ -28,6 +28,12 @@ lowfreq: 0
 highfreq: null
 window: hann
 
+# There are four candidates of `phoneme_dict_path` provided for Chinese as shown below,
+#     1) (default) 24-final Pinyin: "scripts/tts_dataset_files/zh/24finals/pinyin_dict_nv_22.10.txt",
+#     2) IPA converted from 24-final Pinyin: "scripts/tts_dataset_files/zh/24finals/ipa_dict_nv23.05.txt",
+#     3) 36-final Pinyin: "scripts/tts_dataset_files/zh/36finals/pinyin_dict_nv23.05.txt",
+#     4) IPA converted from 36-final Pinyin: "scripts/tts_dataset_files/zh/36finals/ipa_dict_nv23.05.txt"
+# Suggest to choose IPA symbol set converted from 36-final Pinyin because better audio quality were observed.
```

Comment:
If 35-final is the recommended setting, why not use that as the default?

Reply:
good point. I will change to default 36-final IPA.